pages/site-details/[uuid].tsx

```diff
@@ -26,3 +26,5 @@ export const getServerSideProps = withSites({
 });
 
 export default NewSiteDetails;
+
+(NewSiteDetails as any).hideNav = true;
```

Comment:
so there's three ways to handle this case I think:

1. We can leave it the same, which would be easy, wouldn't increase the amount of code we have, and would still be clear enough. Casting things to `any` is really not a best practice when using TypeScript, though, so that would be the tradeoff. 
2. We can add an optional `hideNav` prop to `NewSiteDetailsProps`
3. We can use [`Object.assign()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) to [implicitly add](https://stackoverflow.com/questions/57712682/react-functional-component-static-property#:~:text=With%20implicit%20typing%20(Best%20Solution)) the `hideNav` property (and its inferred boolean type) to our component. 

@AndrewLester @neha-vard what do y'all think would be best? I think the third option is better than the second, but I don't have a strong preference for the third over the first.

Reply:
@AndrewLester's suggestion worked and seems to align with @ericcccsliu's suggestion for site-details/index.tsx